packages/components/collapsible-panel/src/collapsible-panel.tsx

```diff
@@ -245,7 +250,9 @@ const CollapsiblePanel = (props: TCollapsiblePanel) => {
                 <Spacings.Inset scale={scale}>
                   <SectionContent
                     id={panelContentId}
-                    aria-hidden={isOpen ? 'false' : 'true'}
+                    role="region"
+                    aria-labelledby={panelButtonId}
+                    hidden={isOpen}
```

Comment:
Updated attributes based on [this reference](https://w3c.github.io/aria-practices/examples/accordion/accordion.html).